user-interface/frontend/themes/datamanager/components/grid-templates.css

```diff
@@ -20,7 +20,7 @@
 
 .main.raw-data {
   grid-template-columns: minmax(min-content, 80%) minmax(min-content, 20%);
-  grid-template-rows: minmax(min-content, 20%) minmax(min-content, 80%);
+  grid-template-rows: minmax(min-content, 10%) minmax(min-content, 90%);
```

Comment:
What you end up with is the second row always being 90% because you overwrite this in the main.css. You should set this to 90% and only make one row flexible in height.